build_production/us/faq/index.html

```diff
@@ -240,11 +252,59 @@
 				     data-target="#collapse-speaker1" aria-expanded="true" aria-controls="collapse-speaker1">
-						What else should I know?						<i class="fa"></i>
+						Are there any video recording guidelines?						<i class="fa"></i>
 					
 				</div>
 
 				<div id="collapse-speaker1" class="collapse" aria-labelledby="heading-speaker1" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						Yes! We have put together steps on how to record an awesome video with the opens source software OBS. The steps can be found <a href='https://github.com/devconfcz/virtual-presentations/blob/master/Recording.md'>here</a>.					</div>
+				</div>
+			</div>
+					<div class="card">
+				<div class="card-header collapsed mont-600" id="heading-speaker2" data-toggle="collapse" 
+				     data-target="#collapse-speaker2" aria-expanded="true" aria-controls="collapse-speaker2">
+						What are the audio & video requirements for the pre-recorded sessions?						<i class="fa"></i>
+					
+				</div>
+
+				<div id="collapse-speaker2" class="collapse" aria-labelledby="heading-speaker2" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						These are the audio & video requirements: <br><br>Video Format: MP4 <br><br>Video Frame Rate: 30 fps <br><br>Video Frame Size: 960x540 (you can make it higher resolution if you'd like) <br><br>Video Bit Rate: 1000kbps <br><br>Audio Format: AAC <br><br>Audio Channels: 2 <br><br>Audio Sample Rate: 44100 <br><br>Audio Bit Rate: 128kbps					</div>
+				</div>
+			</div>
+					<div class="card">
+				<div class="card-header collapsed mont-600" id="heading-speaker3" data-toggle="collapse" 
+				     data-target="#collapse-speaker3" aria-expanded="true" aria-controls="collapse-speaker3">
+						How should I submit the pre-recorded video?						<i class="fa"></i>
+					
+				</div>
+
+				<div id="collapse-speaker3" class="collapse" aria-labelledby="heading-speaker3" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						We recommend uploading the video to youtube as unlisted. We will send out an email with instructions on how to submit the youtube link of your video. <br><br> If you would prefer not to use youtube, send the organizers an email at <a href='mailto:info@devconf.us'>info@devconf.us</a> and we will work with you to figure out alternate options.					</div>
+				</div>
+			</div>
+					<div class="card">
+				<div class="card-header collapsed mont-600" id="heading-speaker4" data-toggle="collapse" 
+				     data-target="#collapse-speaker4" aria-expanded="true" aria-controls="collapse-speaker4">
+						Will there be speaker coaching?						<i class="fa"></i>
+					
+				</div>
+
+				<div id="collapse-speaker4" class="collapse" aria-labelledby="heading-speaker4" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						Yes! When you confirm your session acceptance, you will be asked whether you would like to have a speaker coach and will be matched with one.					</div>
```

Comment:
are we actually asking?

Reply:
Yes we are, it is in the confirmation form.